autotest/ogr/ogr_pgeo.py

```diff
@@ -366,3 +367,43 @@ def test_ogr_pgeo_11():
                                       max_error=0.0001) != 0:
         feat.DumpReadable()
         pytest.fail('did not get expected geometry')
+
+###############################################################################
+
+
+def test_ogr_pgeo_read_domains():
+
+    ds = gdal.OpenEx('data/pgeo/domains.mdb', gdal.OF_VECTOR)
```

Comment:
you likely need a test of ds is not None like in test_ogr_pgeo_11() to detect misconfigured driver tof fix the test failures